tests/generate-data/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "private": true,
+  "devDependencies": {
+    "@babel/core": "7.5.5",
+    "@babel/preset-env": "7.5.5",
+    "babelify": "10.0.0",
+    "browserify": "16.3.0",
+    "npm-run-all": "^4.1.5",
+    "terser": "4.1.3"
+  },
+  "browserslist": "node 0.12",
+  "scripts": {
+    "build": "browserify --debug -t [ babelify --presets [ @babel/preset-env ] ] dist/foo.js -o foo.js",
+    "minify:inline": "terser foo.js -c --source-map \"content=inline,url=inline\" -o ../data/inline-map.js && rm ../data/inline-map.js.map",
```

Comment:
So `terser` creates a source map file even if `url=inline`. We should report this issue

Reply:
`rm` won't work on windows machines. Please use `rimraf` package